functions/Get-DbaAgentJob.ps1

```diff
@@ -120,7 +120,7 @@ function Get-DbaAgentJob {
             }
             if ($Database) {
                 $jobs = $jobs | Where-Object {
-                    $_.JobSteps.DatabaseName -in $Database
+                    $_.JobSteps.DatabaseName | Where-Object { [string]$_ -in $Database }
```

Comment:
This is already a `Where-Object` why does it need a second call to `Where-Object`?

Reply:
JobSteps.DatabaaeName is something like an aggregate property. I don't find any documentation about this but If you create job with two steps you found what condition in current version dbatools doesn't return this job because excepts first operand in the "-in" operator is value (not an array / collection).

Reply:
Understand that portion but the only change required for this is to change the `-in` to `-Contains`

Reply:
Actually you can reformat that line of code as well from this:
```
            if ($Database) {
                $jobs = $jobs | Where-Object {
                    $_.JobSteps.DatabaseName -in $Database
                }
            }
```

To this:
```
            if ($Database) {
                $jobs = $jobs | Where-Object { $_.JobSteps.DatabaseName -contains $Database }
            }
```

Reply:
I don't think what "contains" can be apply here because $Database is array too.
I can't test it now, but if it is works - okay.

Reply:
Oh, ok...now I follow. Expanded the test to pass in multiple databases to the parameter and issue is it is trying to compare an Object to a String array.